modules/core_project_factory/main.tf

```diff
@@ -74,19 +74,14 @@ resource "null_resource" "preconditions" {
     folder_id        = var.folder_id
     shared_vpc       = var.shared_vpc
   }
-
   provisioner "local-exec" {
-    command = <<EOD
-${path.module}/scripts/preconditions.sh \
-    --credentials_path '${var.credentials_path}' \
-    --impersonate_service_account '${var.impersonate_service_account}' \
-    --billing_account '${var.billing_account}' \
-    --org_id '${var.org_id}' \
-    --folder_id '${var.folder_id}' \
-    --shared_vpc '${var.shared_vpc}'
-EOD
-
-
+    command     = local.pip_requirements_absolute_path
+    interpreter = ["pip3", "install", "-r"]
+    on_failure  = "continue"
+  }
+  provisioner "local-exec" {
```

Comment:
You should always have a newline in between resources.

Reply:
Fixed